cmd/helm/install.go

```diff
@@ -273,6 +273,14 @@ func (i *installCmd) run() error {
 // Merges source and destination map, preferring values from the source map
 func mergeValues(dest map[string]interface{}, src map[string]interface{}) map[string]interface{} {
 	for k, v := range src {
+		// If the new value is literally the string "null", remove the value's key.
+		// This allows helm's multiple value sources (value files or --set) to
```

Comment:
- [ ] s/helm's/Helm's/
- [ ] s/multiple value sources/various sources of values/